build/repo.targets

```diff
@@ -285,7 +294,16 @@
     <PropertyGroup>
         <PublishDir Condition="'$(PublishShare)' != ''">$(PublishShare)\fallbackArchives</PublishDir>
     </PropertyGroup>
-    <Copy SourceFiles="$(RepositoryRoot)artifacts\$(TimestampOutputPackageName).lzma;$(RepositoryRoot)artifacts\$(NoTimestampOutputPackageName).lzma" DestinationFolder="$(PublishDir)" Condition="'$(PublishDir)' != ''" />
+
+  <ItemGroup>
```

Comment:
nit: in fact you don't need this itemgroup and should be able to just specify the files in the Copy target.